tailwind.config.ts

```diff
@@ -58,6 +58,11 @@ export default {
         },
         white: colors.white,
         transparent: colors.transparent,
+        primaryGradient: {
+          first: lightDark(0.7985, 0.132, 159.5), // #67D79F
+          middle: lightDark(0.7292, 0.127, 214.3), // #10BBD8
+          last: lightDark(0.7617, 0.119, 184.67), // #3DCABB
+        },
         primary: lightDark(0.68, 0.153, 160), // #00B578
```

Comment:
Do we still have primary ?

Reply:
Still used for text, borders, input and label backgrounds and others